src/scripts/main.js

```diff
@@ -1,3 +1,32 @@
 'use strict';
 
-// write code here
+const tree = document.querySelector('.tree');
+const lis = tree.querySelectorAll('li');
```

Comment:
Similar to the previous comment, ensure that `tree` is not `null` before querying for 'li' elements. Otherwise, this will throw an error.